stdlib/LinearAlgebra/test/qr.jl

```diff
@@ -214,6 +215,15 @@ rectangularQ(Q::LinearAlgebra.AbstractQ) = Matrix(Q)
                 @test_throws DimensionMismatch mul!(Vector{eltya}(undef, n+1), q, b)
             end
         end
+
+        @testset "Test nullspace (issue #54354)" begin
+            a15null = nullspace(qr(a[:,1:n1], ColumnNorm()))
+            @test rank([a[:,1:n1] a15null]) == 10
```

Comment:
Is it always 10, or does it depend on the value of `n1`?

Reply:
Well, it should always be `n`, since `n1 = div(n, 2) < n = 10`. However, when I looked at the counterpart in `test/dense.jl` it tested for equality to 10. Perhaps `@test rank([a[:,1:n1] a15null]) == n` would be better?